src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DampedHarmonic.cpp

```diff
@@ -390,7 +390,7 @@ void damped_harmonic_impl(
   //  0.5 * n^a n^b Pi_{ab}
   //  0.5 * n^a n^b Phi_{iab}
   // so we reuse that work by taking them as arguments.
-  get<0>(d4_muS_over_lapse) *= -get(mu_S);
+  get<0>(d4_muS_over_lapse) *= -get(mu_S) * get(one_over_lapse);
```

Comment:
Don't multiply by one over lapse here. I think you can just remove the multiplication of lapse on line 367 above.

Reply:
That would be consistent with the comment above line 364. But the value is used in line 372, so that needs a factor of the lapse then.

Reply:
Oops missed that. @geoffrey4444 could you fix the comment on line 361 then?

Reply:
The equation in that comment was correct but not what was actually being computed, so I updated it to reflect that here we're computing d_t lapse, not dt_lapse / lapse. Squashed